recipes/testpath/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "testpath" %}
+{% set version = "0.3" %}
+{% set wheel_tag = "py2.py3-none-any" %}
+{% set fn = "{}-{}-{}.whl".format(name, version, wheel_tag) %}
+{% set sha256 = "f16b2cb3b03e1ada4fb0200b265a4446f92f3ba4b9d88ace34f51c54ab6d294e" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ fn }}
+  url: https://pypi.io/packages/py2.py3/{{ name[0] }}/{{ name }}/{{ fn }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps {{ fn }}
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - pathlib2  # [py2k]
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - testpath
+
+about:
+  home: https://testpath.readthedocs.io
+  license: MIT
+  license_family: MIT
+  # FIXME: flit wheels don't include license
+  # license_file
```

Comment:
Interesting. Would like to discuss this more. Feel free to direct me to the right forum (i.e. probably not this PR).